src/interface/report/PlayerLoader.tsx

```diff
@@ -237,154 +274,146 @@ class PlayerLoader extends React.PureComponent<Props, State> {
         </Panel>
       </div>
     );
-  }
+  };
 
-  render() {
-    const { report, fight, playerName, playerId } = this.props;
-
-    const error = this.state.error;
-    if (error) {
-      return this.renderError(error);
-    }
+  if (error) {
+    return renderError(error);
+  }
 
-    if (report.gameVersion === 2) {
-      return this.renderClassicWarning();
-    }
+  if (selectedReport.gameVersion === 2 || selectedReport.gameVersion === 3) {
```

Comment:
this should use something more clear than checking if gameVersion is 2 or 3. This is actually checking `isClassic(selectedReport)` right? (I don't think that method exists)